catalog.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
```

Comment:
```suggestion
<html class="page" lang="ru">
```
Неиспользуемые классы нужно удалить